src/Select.js

```diff
@@ -598,11 +600,24 @@ var Select = React.createClass({
 
 		}, this);
 
-		return ops.length ? ops : (
-			<div className="Select-noresults">
-				{this.props.asyncOptions && !this.state.inputValue ? this.props.searchPromptText : this.props.noResultsText}
-			</div>
-		);
+		if (ops.length) {
+			return ops;
```

Comment:
```
if (ops.length) return ops

// continue as normal, no else
```

Don't bother with the `else` if you have an early exit `return`.
It is implied.